### DISCUSSION TITLE
Vercel Hosting issue SSL Problem

### DISCUSSION THREAD

**Discussion by codecygen** [vercel/community#5110] [Help]
### Summary

Hello,

Thank you for your assistance and I would like to tell I have been hosting a hobby website in Vercel which is hosted at
https://vercel-083-next-js-blog-website.vercel.app/
but lately, when I try to access the website, it gave me the following error.
Error code: SSL_ERROR_RX_RECORD_TOO_LONG

I tried hosting the website in a secondary project but unfortunately it also failed to connect
https://aras-next-js-blog-website.vercel.app/

I found a potential solution on Vercel's website,
https://vercel.com/docs/cli/certs

I tried issuing ssl certificate with the command
vercel certs ls
Vercel CLI 32.6.1
> No certificates found under username-emailcom [258ms]

vercel certs issue "[vercel-083-next-js-blog-website.vercel.app](http://vercel-083-next-js-blog-website.vercel.app/)"
Vercel CLI 32.6.1
Error: You don't have permissions to access [vercel-083-next-js-blog-website.vercel.app](http://vercel-083-next-js-blog-website.vercel.app/).

I thought SSL is auto issued by Vercel. I had no such issues until yesterday.

I don't really know what to do and need your help.

Thank you.

### Steps to Reproduce

Go to 

https://aras-next-js-blog-website.vercel.app/

and

https://aras-next-js-blog-website.vercel.app/

see error
Error code: SSL_ERROR_RX_RECORD_TOO_LONG

**Comment by amyegan** [comment]
Thanks for reporting this. We had a recent TLS incident that is resolved now: https://www.vercel-status.com/incidents/sfrbn1scg6wn

Please let me know if you continue to have trouble with your sites.

**Reply by codecygen** [reply]
Thank you for resolving the issue however my websites are still down even though the page you provided claims that the incident has been resolved.

What should I do?
Thank you.

**Reply by amyegan** [reply]
Thanks for letting me know. The problem may instead be related to an ISP level block. You can find more info on that in  https://github.com/vercel/community/discussions/5102

**Reply by codecygen** [reply]
Yes it is indeed due to the ISP block as I can access my website with a VPN. However, what I don't understand is, I can still access Vercel's official webpage. That's really odd.
